File: Python3/Secuenciales/ej8/ej8.py
# -*- coding: utf-8 -*-
#Un vendedor recibe un sueldo base mas un 10% extra 
#por comisión de sus ventas, el vendedor desea saber cuanto 
#dinero obtendrá por concepto de comisiones por las tres ventas 
#que realiza en el mes y el total que recibirá en el mes 
#tomando en cuenta su sueldo base y comisiones.

#pedir sueldo base
#almacenar sueldo base
#pedir precio producto
#almacenar precio producto
#comision = (precioProducto*0.10)*3
#totalMes = sueldoBase+comision

print("==..::Calcular comisión::..==")
sueldoBase = float(input("Introduce el sueldo base> "))
precioProducto = float(input("Introduce el precio del producto> "))
comision = (precioProducto*0.10)*3
totalMes = sueldoBase+comision
print("El vendedor este mes recibira un total de "+str(totalMes)+" del cúal por comisiones es "+str(comision))

File: Java/Secuenciales/ej8.java
import java.util.Scanner;

/**
* Un vendedor recibe un sueldo base mas un 10% extra por comisión 
* de sus ventas, el vendedor desea saber cuanto dinero obtendrá por concepto 
* de comisiones por las tres ventas que realiza en el mes y el total que recibirá 
* en el mes tomando en cuenta su sueldo base y comisiones.
*/

//Almacenar sueldoBase
//Almacenar valor venta1
//Almacenar valor venta2
//Almacenar valor venta3
//Calcular total = sueldoBase + (venta1*0.10+venta2*0.10+venta3*0.10)

public class ej8 {
	public static void main(String[] args) {
		double sueldoBase;
		double venta1;
		double venta2;
		double venta3;
		double total;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner
		
		System.out.println("Introduce el sueldo base> ");
		sueldoBase = scanner.nextDouble();
		
		System.out.println("Introduce la primera venta del mes> ");
		venta1 = scanner.nextDouble();
		
		System.out.println("Introduce la segunda venta del mes> ");
		venta2 = scanner.nextDouble();
		
		System.out.println("Introduce la tercera venta del mes> ");
		venta3 = scanner.nextDouble();
		
		total = sueldoBase + (venta1*0.10+venta2*0.10+venta3*0.10);
		
		System.out.println("El total de ganancia en el mes es "+total);
		
	}
}